renovate.json

```diff
@@ -41,7 +41,12 @@
     "addLabels": ["lang: nodejs"]
   },
   "docker": {
-    "pinDigests": true
+    "pinDigests": true,
+    "ignorePaths": [
+      "release/**",
+      "kustomize/**",
+      "kubernetes-manifests/**"
+    ]
```

Comment:
**Thought:**
Renovate shouldn't touch `/kustomize/base/**` and `kubernetes-manifests/**` at all.
I wonder if we're able to move out of the `"kubernetes"` configuration's `ignorePath`.